.github/ISSUE_TEMPLATE/other-issue.yml

```diff
@@ -0,0 +1,23 @@
+name: General issue report
+description: File an issue report
+body:
+  - type: checkboxes
+    id: checklist
+    attributes:
+      label: Answers checklist.
+      description: Before submitting a new issue, please follow the checklist and try to find the answer.
+      options:
+        - label: I have read the component documentation [ESP-IDF Components](https://components.espressif.com) and the issue is not addressed there.
```

Comment:
This sentence is not very clear — components.espressif.com itself is not "component documentation". Maybe something like this?
```suggestion
        - label: I have read the documentation of the component in question and the issue is not addressed there.
```

(applies to the other places where this checkbox is used)